content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -59,6 +59,17 @@ firewall-cmd --permanent --zone=trusted --add-source=10.43.0.0/16 #services
 firewall-cmd --reload
 ```
 
+You should also add [port rules](https://firewalld.org/documentation/howto/open-a-port-or-service.html)
+for all the [RDI services]({{< relref "/integrate/redis-data-integration/ingest/reference/ports" >}})
+you intend to use:
+
+```bash
+firewall-cmd --permanent --add-port=8080/tcp  # (Required) rdi-operator/rdi-api
+firewall-cmd --permanent --add-port=9090/tcp  # vm-dis-reloader
+firewall-cmd --permanent --add-port=9092/tcp  # prometheus-service
+firewall-cmd --permanent --add-port=9121/tcp  # rdi-metric-exporter
+```
+
 You may also need to open other ports if your setup requires them. See the K3s
 [Inbound rules](https://docs.k3s.io/installation/requirements?_highlight=red&_highlight=hat&os=rhel#inbound-rules-for-k3s-nodes)
 docs for more information. If you change the default CIDR for pods or services,
```

Comment:
I don't think we give the user the option to change the default CIDR for pods and services